pypeit/data/arc_lines/lists/OH_MOSFIRE_Y_lines.dat

```diff
@@ -0,0 +1,182 @@
+# Created by DP on 2022-Apr-13. R=3388 (from MOSFIRE website) and dv=40km/s
```

Comment:
To all of these files can you explicitly state that the wavelengths are vacuum in the README.